filesystem/minifs.ksy

```diff
@@ -0,0 +1,112 @@
+meta:
+  id: minifs
```

Comment:
`tplink_minifs`? Maybe it helps someone to include `tplink` in there to find the format they need, if the format is TP-Link specific.

Reply:
This makes sense.